static/docker-compose.yml

```diff
@@ -80,6 +80,42 @@ services:
     volumes:
     - letsencrypt:/etc/letsencrypt:rw
 
+  notification-engine:
+    image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    depends_on:
+      - plextracdb
+      - redis
+    restart: always
+    environment:
+      CB_API_PASS: "${CB_API_PASS:?err}"
+      CB_API_USER: "${CB_API_USER:?err}"
+      COUCHBASE_URL: "http://plextracdb"
+      LOG_LEVEL: "${LOG_LEVEL:-error}"
+      REDIS_CONNECTION_STRING: "${REDIS_CONNECTION_STRING:-redis}"
+      REDIS_PASSWORD: "${REDIS_PASSWORD:?err}"
```

Comment:
@evanhsu I think we need CLIENT_DOMAIN_NAME to be required for the Engine here too because the Engine is responsible for constructing the content for the emails, and therefore will need the domain name to create the urls.

Reply:
Disregard. I got confused. Just double checked the code and it is only the notification-sender service that actually needs this env var.